### DISCUSSION TITLE
I install codeChecker in docker , how to check my project and add new product?

### DISCUSSION THREAD

**Discussion by zhuxiu1234** [Ericsson/codechecker#3482] [Q&A] [answered]
I install codeChecker in docker , how to check my project and add new product?

**Comment by csordasmarton** [comment, marked as answer]
@zhuxiu1234 You can find information how to add a new product here: https://github.com/Ericsson/codechecker/blob/master/docs/web/products.md

Our [official docker image](https://hub.docker.com/r/codechecker/codechecker-web) is only for running a CodeChecker server. If you want to analyze/check your project you have to extend this image or install CodeChecker to your host system.  You can find our installation guide here: https://github.com/Ericsson/codechecker#install-guide

We also have a howto guide which helps you to understand how to use CodeChecker: https://github.com/Ericsson/codechecker/blob/master/docs/usage.md
